packages/back-end/types/experiment-snapshot.d.ts

```diff
@@ -164,6 +166,25 @@ export interface ExperimentSnapshotInterface {
   unknownVariations: string[];
   multipleExposures: number;
   analyses: ExperimentSnapshotAnalysis[];
+
+  health?: ExperimentSnapshotHealth;
+}
+
+export interface ExperimentSnapshotHealth {
+  traffic: ExperimentSnapshotTraffic;
+}
+
+export interface ExperimentSnapshotTraffic {
+  overall: ExperimentSnapshotTrafficDimension[];
```

Comment:
does overall need to be an array? My understanding is that we will always be accessing the first index to get the Experiment Balance Check table values so maybe we can just make it a single object, but I may be missing something.

Reply:
It doesn't need to be. It just made it easier to create an empty object without letting overall be undefined. I'll change it.